mateo-mustapic/assignments/liferay-app/sdk/portlets/todo-portlet/docroot/WEB-INF/src/content/Language.properties

```diff
@@ -7,7 +7,9 @@ contact-us=Contact us
 date-of-birth=Date of birth
 deleted-tasks-do-not-appear-in-history=Deleted tasks do not appear in history
 enter-task=Enter Task
-error=Error, form not submitted, you didn't fill out all the fields!
+error=Error, form not submitted, you didn't fill out all the fields
+form-submission-failed=Form submission failed
+form-successfully-submitted=Form successfully submitted.
```

Comment:
@mateomustapic here you can make great User Experience improvement. In Liferay there is abstract message **Your request succeeded** or **Your request failed**.. I guess @markocikos will be happy with Liferay default BUT in my experience, un-technical user is dummy enough to not know neither what is form neither what is Request :) I would be happy with something **Requested operation couldn't be completed** and **Requested operation completed** HINT: Sometimes if I have no idea I observe how google in gmail notifies user and what language they use.